pdal/SpatialReference.cpp

```diff
@@ -485,7 +485,7 @@ std::string SpatialReference::getWKT1() const
     if (srs)
     {
         char *buf = nullptr;
-        const char* apszOptions[] = { "FORMAT=WKT1_GDAL", nullptr };
+        const char* apszOptions[] = { ["FORMAT=WKT1_GDAL", "ALLOW_ELLIPSOIDAL_HEIGHT_AS_VERTICAL_CRS=YES"], nullptr };
```

Comment:
This is invalid C++. You'll have to drop the `[ ]` in order for it to work.

Reply:
In my docker build i did the following on top of master
```
  sed -i s/"FORMAT=WKT1_GDAL"/["FORMAT=WKT1_GDAL","ALLOW_ELLIPSOIDAL_HEIGHT_AS_VERTICAL_CRS=YES"]/g   pdal/SpatialReference.cpp ;\
```
and it successfully built and ran correctly before i created the PR

Reply:
Not sure what to tell you, but that's not valid C++. All the CI tests fail. Just remove the `[ ]`.

C:\PROGRA~2\MICROS~1\2019\ENTERP~1\VC\Tools\MSVC\1428~1.293\bin\Hostx64\x64\cl.exe  /nologo /TP -DLASZIP_DLL_IMPORT=1 -DPDAL_DLL_EXPORT=1 -DWIN32_LEAN_AND_MEAN -DWINDOWS -D_CRT_NONSTDC_NO_WARNING -D_CRT_SECURE_NO_DEPRECATE -D_CRT_SECURE_NO_WARNINGS -D_SCL_SECURE_NO_WARNINGS -Dpdalcpp_EXPORTS -I..\vendor\gtest\include -I..\vendor\gtest -I..\ -Iinclude -I..\vendor -I..\vendor\eigen -I..\vendor\pdalboost -I..\vendor\nlohmann -IC:\Miniconda\envs\test\Library\include /DWIN32 /D_WINDOWS /W3 /GR /EHsc /MD /Zi /O2 /Ob1 /DNDEBUG /wd4068 /wd4290 /wd4800 /wd4267 /wd4250 /EHsc /MP2 /showIncludes /FoCMakeFiles\pdalcpp.dir\pdal\SpatialReference.cpp.obj /FdCMakeFiles\pdalcpp.dir\ /FS -c ..\pdal\SpatialReference.cpp
..\pdal\SpatialReference.cpp(488): error C2143: syntax error: missing ']' before 'string'
..\pdal\SpatialReference.cpp(488): error C3260: 'string': skipping unexpected token(s) before lambda body
..\pdal\SpatialReference.cpp(493): error C3493: 'wkt1' cannot be implicitly captured because no default capture mode has been specified
..\pdal\SpatialReference.cpp(493): error C3493: 'buf' cannot be implicitly captured because no default capture mode has been specified